www_src/pages/section-2/add-page.jsx

```diff
@@ -0,0 +1,15 @@
+var React = require('react');
+
+module.exports = React.createClass({
+  render: function () {
+    return (
+      <button
+        onClick={this.handleClick}
```

Comment:
`onClick={this.props.onClick}`? There doesn't seem to be a need for the pass-through